src/common/blockchain.sh

```diff
@@ -237,25 +237,28 @@ function install_fabric_chaincode {
     local CC_PATH=$3
     local CC_TYPE=$4
 
-    local CHAINCODE_FILES
-
     echo "Installing chaincode '$CC_PATH' with id '$CC_ID' and version '$CC_VERSION'..."
     
-    CHAINCODE_FILES=$(find "$CC_PATH" -type f ! -name "*test*")
-    CHAINCODE_FILE_OPTS=""
-    for CHAINCODE_FILE in ${CHAINCODE_FILES}
-    do
-        CHAINCODE_FILE_OPTS="${CHAINCODE_FILE_OPTS} -F files[]=@${CHAINCODE_FILE}"
-    done
+    # Cannot leave behind folders... for instance, a chaincode component
+    # may have additional files such as CouchDB index files.
+    # Hence, including all folders and files
+    pushd "$CC_PATH"
+    CC_ZIP_FILE="${CC_ID}.zip"
+    echo "Creating ZIP file for chaincode: ${CC_ZIP_FILE}"
+    zip -r "$CC_ZIP_FILE" ./*
```

Comment:
@rolivieri  Wondering about how "standard" is the zip command and whether this command available in the OS image used where the pipelines run.

Reply:
@jorgedr94 It is available; this has been tested and the command is available in the docker image used in the pipelines.